config/full_node.json

```diff
@@ -0,0 +1,101 @@
+{
+    "parent-chain": {
+        "connection": {
+            "url": "wss://arbitrum-sepolia.infura.io/ws/v3/"
+        }
+    },
+    "chain": {
+        "id": 12331319,
+        "info-files": [
+            "/config/l2_chain_info.json"
+        ]
+    },
+    "node": {
+        "staker": {
+            "parent-chain-wallet": {
+                "private-key": ""
+            },
+            "disable-challenge": false,
+            "enable": true,
+            "staker-interval": "10s",
+            "make-assertion-interval": "10s",
+            "strategy": "MakeNodes"
+        },
+        "sequencer": true,
+        "dangerous": {
+            "no-sequencer-coordinator": true,
+            "disable-blob-reader": true
+        },
+        "parent-chain-reader": {
+            "poll-only": true,
+            "poll-interval": "60s"
+        },
+        "delayed-sequencer": {
+            "enable": true
+        },
+        "seq-coordinator": {
+            "enable": false,
+            "redis-url": "redis://redis:6379",
+            "lockout-duration": "30s",
+            "lockout-spare": "1s",
+            "my-url": "",
+            "retry-interval": "0.5s",
+            "seq-num-duration": "24h0m0s",
+            "update-interval": "3s"
+        },
+        "batch-poster": {
+            "enable": true,
+            "redis-url": "",
+            "l1-block-bound": "ignore",
+            "hotshot-url": "https://query.decaf.testnet.espresso.network/v0",
```

Comment:
@jparr721  I think Charles wanted this to be mainnet. So I have asked in Zulip Mat/Robert to deploy a light client address thats compatible with mainnet prover

Reply:
I've changed 'testnet' to 'mainnet' in the url. Also added a placeholder for the light client address ti be changed.